pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/PatchFormAnswersRouteStandaloneTest.java

```diff
@@ -1852,10 +1852,10 @@ public void testPatch_fileAnswer_createAndUpdate() {
         assertEquals(guid, answer.get().getAnswerGuid());
         assertEquals(stableId, answer.get().getQuestionStableId());
         assertEquals(QuestionType.FILE, answer.get().getQuestionType());
-        assertEquals(upload1.getFileName(), ((FileAnswer)answer.get()).getValue().getFileName());
-        assertEquals(upload1.getFileSize(), ((FileAnswer)answer.get()).getValue().getFileSize());
+        assertEquals(upload1.getFileName(), ((FileAnswer)answer.get()).getValue().get(0).getFileName());
```

Comment:
We can extend FileAnswer with getInfo(int index) method. This will simplify the usage. Instead of .getValue().get(i) we will use .getInfo(i). And inside this method we can easily introduce not null check